doc/source/ray-air/doc_code/checkpoint_usage.py

```diff
@@ -0,0 +1,127 @@
+# __checkpoint_quick_start__
+from ray.train.tensorflow import to_air_checkpoint
+
+
+# This can be a trained model.
+def build_model() -> tf.keras.Model:
+    model = tf.keras.Sequential(
+        [
+            tf.keras.layers.InputLayer(input_shape=(1,)),
+            tf.keras.layers.Dense(1),
+        ]
+    )
+    return model
+
+model = build_model()
+
+checkpoint = to_air_checkpoint(model)
+# __checkpoint_quick_end__
+
+
+import ray
+import pandas as pd
+from sklearn.datasets import load_breast_cancer
+from sklearn.model_selection import train_test_split
+
+from ray.data.preprocessors import *
+
+data_raw = load_breast_cancer()
+dataset_df = pd.DataFrame(data_raw["data"], columns=data_raw["feature_names"])
+dataset_df["target"] = data_raw["target"]
+train_df, test_df = train_test_split(dataset_df, test_size=0.3)
+train_dataset = ray.data.from_pandas(train_df)
+valid_dataset = ray.data.from_pandas(test_df)
```

Comment:
Should we include this in the doc so the user can run the code directly and play around with the checkpoint themselves?